gaia-list.js

```diff
@@ -256,15 +256,6 @@ module.exports = component.register('gaia-list', {
       width: auto;
     }
 
-    /**
-     * Reverse the icons when the document is RTL mode
-     */
-
-    :host-context([dir=rtl]) ::content i:before {
-      transform: scale(-1, 1);
-      text-align: end;
-    }
-
```

Comment:
So there is 3 levels here: gaia-icon, gaia-list, the apps that uses gaia-list. The question is: where do we put direction-specific rules? I guess it makes sense to put them in gaia-list, but as we will have a lot of occurrences of these, wouldn't it better to put them in gaia-icons? (TBH I really don't know). 

An advantage I see in putting them in gaia-icons is that we stay compatible with consumers. On the other hand, it would bloat a bit gaia-icons. WDYT?